aip/firebase/3260.md

```diff
@@ -0,0 +1,46 @@
+---
+aip:
+  id: 3260
+  scope: firebase
+  state: draft
+  created: 2020-06-10
+  updated: 2019-06-10
+permalink: /firebase/3260
+redirect_from:
+  - /3260
+---
+
+# Firebase CLI
+
+## Guidance
+
+### Project Selection
+
+The Firebase CLI works with "project directories" that can be associated with
+one or more Firebase projects via the `use` command. This ambient project
+selection can be overridden by passing the `-P` or `--project` flag.
+
+All Firebase CLI commands **must** determine the intended project for a command
+based on common project selection, except when a command involves multiple
+projects (e.g. a command that clones configuration from one project to
+another).
```

Comment:
We also have commands that are project agnostic.  For example, `firebase ext:dev:init`.